automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodImportSpec.scala

```diff
@@ -21,42 +21,46 @@ class MethodImportSpec extends FreeSpec with Matchers with WebBrowserSpec with W
               api.methodConfigurations.createMethodConfigInWorkspace(billingProject, sourceWorkspaceName,
                 method, method.methodNamespace, method.methodName, 1, Map.empty, Map.empty, method.rootEntityType)
 
-              withSignIn(user) { listPage =>
-                val methodConfigTab = listPage.enterWorkspace(billingProject, destWorkspaceName).goToMethodConfigTab()
+              withWebDriver { implicit driver =>
```

Comment:
`withWebDriver` will take a blank page screenshot when API fails. This trait should be called immediately before the SignIn so the screenshot is taken on failed UI page.